.circleci/run.sh

```diff
@@ -127,20 +130,12 @@ coverage()
     rm -rf test/compilable/issue17167.sh
 
     # rebuild dmd with coverage enabled
-    # use the just build dmd as host compiler this time
-    local build_path=generated/linux/$BUILD/$MODEL
-    # `generated` gets cleaned in the next step, so we create another _generated
-    # The nested folder hierarchy is needed to conform to those specified in
-    # the generate dmd.conf
-    mkdir -p _${build_path}
-    cp $build_path/dmd _${build_path}/host_dmd
-    cp $build_path/dmd.conf _${build_path}
-    make -j$N -C src -f posix.mak MODEL=$MODEL BUILD=$BUILD HOST_DMD=../_${build_path}/host_dmd PIC="$PIC" clean
-    make -j$N -C src -f posix.mak MODEL=$MODEL BUILD=$BUILD HOST_DMD=../_${build_path}/host_dmd ENABLE_COVERAGE=1 ENABLE_WARNINGS=1 PIC="$PIC"
+    _${build_path}/host_dmd -run src/build.d MODEL=$MODEL BUILD=$BUILD HOST_DMD=$PWD/_${build_path}/host_dmd ENABLE_COVERAGE=1
```

Comment:
Why does it use the generated compiler to compile `build.d` here?

Reply:
IIRC it was because https://github.com/dlang/installer/pull/297 was never merged and it's easier to access the generated compiler here.

Reply:
Guess we could also precompile `build.d` as done in the makefiles (saves some memory and is less redundant if we want to compile with `-g`)

Reply:
Yeah good point, but I don't think it's that important (considering all the other tasks performed here).
Anyhow, sth. for a different PR ;-)

Reply:
Agreed, that belongs in another PR.